test/parallel/test-stdout-buffer-flush-on-exit.js

```diff
@@ -15,13 +15,11 @@ if (process.argv[2] === 'child') {
   process.exit();
 }
 
-[22, 21, 20, 19, 18, 17, 16, 16, 17, 18, 19, 20, 21, 22].forEach((exponent) => {
+[22, 16].forEach((exponent) => {
```

Comment:
>  However, if stdout is not being flushed on exit, it will not fail deterministically. It will fail most of the time.

Yes, that was the intention. If stdio is not flushed the new test in this PR should fail.

We're likely in agreement but our messaging is probably being misinterpreted between us.